ui/tests/integration/utils/client-count-utils-test.js

```diff
@@ -7,595 +7,285 @@ import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import {
   filterVersionHistory,
-  flattenDataset,
```

Comment:
In a follow on PR I've refactored to no longer need this helper, deleting the test here